src/modules/buttons/Control/Control.js

```diff
@@ -9,10 +9,7 @@ import './Control.css';
 const b = block('j-control');
 
 const propTypes = {
-    icon: PropTypes.oneOf([
-        'add',
-        'remove',
-    ]).isRequired,
+    icon: Icon.propTypes.type,
```

Comment:
Unnecessary and unobvious. The previous version not only for type checking but also serves as some documentation